docs/theme/components/collocator/components/input-number.tsx

```
@@ -0,0 +1,41 @@
+    sign,
+  })
+
+  const changeFn = (v?: string) => {
+    setValue(v)
+    setSign(false)
+    const tempRelate = Number(v) && item.related ? item.related : {}
+    
+    if (parent) {
+      const tempConfig = Number(v) && item.mergeRelated && isObject(item.mergeRelated) ? Object.keys(item.mergeRelated).reduce((pre, cur) => (
+        {
+          ...pre,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)